src/App.jsx

```diff
@@ -14,57 +15,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, selectedGood] = useState('Jam');
```

Comment:
Consider renaming `selectedGood` to `setSelectedGood` to follow the React convention for state setter functions. This will make it clearer that this function is used to update the state.